forms-flow-bpm/src/main/resources/application.yaml

```diff
@@ -182,4 +182,10 @@ logging:
     org.springframework.security: ${CAMUNDA_APP_ROOT_LOG_FLAG}
     org.springframework.web: ${CAMUNDA_APP_ROOT_LOG_FLAG}
     org.springframework.jdbc: ${CAMUNDA_APP_ROOT_LOG_FLAG}
-    org.camunda.bpm: ${CAMUNDA_APP_ROOT_LOG_FLAG}
\ No newline at end of file
+    org.camunda.bpm: ${CAMUNDA_APP_ROOT_LOG_FLAG}
+  pattern:
+    console: '%d{yyyy-MM-dd HH:mm:ss} - %msg%n'
+    file: '%d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} - %msg%n'
+  file:
+    name: error.log
```

Comment:
may be change the name to just bpmlogs or something generic as logs would contain any log level based on configuration ?

Reply:
yes done.